src/pages/Main/HotSeller.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { API } from '../../config';
+import axios, { get } from 'axios';
+
+const HotSeller = () => {
+  const [hotSeller, sethotSeller] = useState([]);
+  const history = useHistory();
+  useEffect(() => {
+    axios
+      .get(API.SELLERLST + `?view_type=order`)
+      .then(result => {
+        sethotSeller(result.data.seller);
+      })
+      .catch(error => {
+        console.log(error.message);
+      });
+  }, []);
+  return (
+    <SellerWrapper>
+      {hotSeller?.map((item, index) => (
```

Comment:
- 코드 사이에 한칸 씩 띄워주시면 더 가독성이 좋아질 것 같은데 어떻게 생각하시나요?
```suggestion
const HotSeller = () => {
  const [hotSeller, sethotSeller] = useState([]);
  const history = useHistory();
  
  useEffect(() => {
    axios
      .get(API.SELLERLST + `?view_type=order`)
      .then(result => {
        sethotSeller(result.data.seller);
      })
      .catch(error => {
        console.log(error.message);
      });
  }, []);
  
  return (
    <SellerWrapper>
      {hotSeller?.map((item, index) => (
```

Reply:
아주 좋다고 생각합니다!! 훨씬 정리되어보이네요